packages/backend/src/apps/postman/actions/send-transactional-email/index.ts

```diff
@@ -30,17 +31,16 @@ export default defineAction({
     if (!result.success) {
       throw fromZodError((result as SafeParseError<unknown>).error)
     }
+    let recipients = result.data.destinationEmail.slice(+progress)
+    recipients = await getRatelimitedRecipientList(recipients)
+    const newProgress = +progress + recipients.length
```

Comment:
nit: maybe would be easier to combine these into a single function:

```suggestion
    const [recipients, newProgress] = await getRatelimitedRecipientList(recipients, currProgress)
```